File: java/8 kyu/CountSheep.java

// https://www.codewars.com/kata/5b077ebdaf15be5c7f000077
import java.util.stream.IntStream;

class CountSheep {
  public static String countingSheep(int num) {
    return String.join("", IntStream.rangeClosed(1, num).boxed().map(n -> String.valueOf(n + "sheep...")).toList());
  }
}

File: c/8kyu/count_sheep.c
//https://www.codewars.com/kata/54edbc7200b811e956000556/
#include <stdbool.h>
#include <stddef.h>

size_t count_sheep(const bool sheep[/* count */], size_t count)
{ 
  int sheeps = 0;
  while(count --){
    if(sheep[count]) sheeps ++;
  }
  return sheeps;
}
